code/game/objects/items/storage/belt.dm

```diff
@@ -231,6 +233,7 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/spaceacillin(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/hypervene(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/quickclot(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus(src)
```

Comment:
Holdon whats this this isnt a fix

Reply:
This was the original state of the belt, when peridaxon was removed, a standard peridaxon injector was too, having been replaced with a QC injector.

Reply:
As for 'Peridaxon+ is rare'...You can just mab 50 phoron sheets from the engine room.

Reply:
Still don't understand how it's a fix ?